src/App.tsx

```diff
@@ -55,6 +57,34 @@ loader.init().then((monaco) => {
   })
 })
 
+const getAppDataFromSearchParams = () => {
+  const urlParams = new URLSearchParams(window.location.search)
+  const encAppData = urlParams.get('appData')
```

Comment:
Not sure if appData is the best choice. Maybe just data or content?

Reply:
`context` is also another option maybe. But i'm fine with appData too honestly

Reply:
Don't remember why I wrote `appData` instead of just `data` or `content`.

Would opt for `data`, why not?

Not `state` to avoid mix with react stuff on the code.

Reply:
I have no strong opinion on that, but appData gives an additional information the data refers to entire application.
data is very generic and we can expect to find it everywhere for other circumstances. You had a great idea. Thank you Davide 😂

Reply:
What about `app_data` or `app-data` then?

Camel case in URL sucks, wrote this way just to quickly test it w/o thinking.

With reference to https://stackoverflow.com/questions/10302179/hyphen-underscore-or-camelcase-as-word-delimiter-in-uris looks like we should use hypens (it's what google recommends too, see third answer).